support-frontend/app/controllers/AuthCodeFlowController.scala

```diff
@@ -61,7 +61,10 @@ class AuthCodeFlowController(cc: ControllerComponents, authService: AsyncAuthent
       "prompt" -> "none",
     )
     val authorizeUrl = s"${config.oauthAuthorizeUrl}?${toQuery(queryParams)}"
-    Redirect(authorizeUrl).addingToSession(SessionKey.state -> state, SessionKey.codeVerifier -> codeVerifier)
+    Redirect(authorizeUrl).addingToSession(
+      SessionKey.state -> state,
+      SessionKey.codeVerifier -> codeVerifier,
+    )
```

Comment:
Would it be much work to put the formatting changes in a separate commit? It’s not super important, but if it’s easy I’d prefer it.

Reply:
Thanks!